dev.shaduri.GSmartControl.yml

```
@@ -0,0 +1,171 @@
+        buildsystem: meson
+        config-opts:
+          - '-Dbuild-examples=false'
+        cleanup:
+          - '*.la'
+          - /include/glibmm-2.4
+          - /include/giomm-2.4
+          - /lib/glibmm-2.4
+          - /lib/giomm-2.4
+          - /lib/libglibmm_generate_extra_defs*
+          - /lib/pkgconfig
+      - name: cairomm
+        sources:
+          - sha256: 4749d25a2b2ef67cc0c014caaf5c87fa46792fc4b3ede186fb0fc932d2055158
+            type: archive
+            url: 'https://www.cairographics.org/releases/cairomm-1.14.4.tar.xz'
+            x-checker-data:
+              type: anitya
+              project-id: 7959
+              url-template: 'https://www.cairographics.org/releases/cairomm-$version.tar.xz'
+              versions:
+                <: '1.15'
+        buildsystem: meson
+        config-opts:
+          - '-Dbuild-examples=false'
+        cleanup:
+          - '*.la'
+          - /include/cairomm-1.0
+          - /lib/cairomm-1.0
+          - /lib/pkgconfig
+      - name: pangomm
+        sources:
+          - sha256: 410fe04d471a608f3f0273d3a17d840241d911ed0ff2c758a9859c66c6f24379
+            type: archive
+            url: 'https://download.gnome.org/sources/pangomm/2.46/pangomm-2.46.3.tar.xz'
+            x-checker-data:
+              type: gnome
+              name: pangomm
+              versions:
+                <: '2.47'
+        buildsystem: meson
+        cleanup:
+          - '*.la'
+          - /include/pangomm-1.4
+          - /lib/pangomm-1.4
+          - /lib/pkgconfig
+      - name: atkmm
+        sources:
+          - sha256: 7c2088b486a909be8da2b18304e56c5f90884d1343c8da7367ea5cd3258b9969
+            type: archive
+            url: 'https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.3.tar.xz'
+            x-checker-data:
+              type: gnome
+              name: atkmm
+              versions:
+                <: '2.35'
+        buildsystem: meson
+        cleanup:
+          - '*.la'
+          - /include/atkmm-1.6
+          - /lib/atkmm-1.6
+          - /lib/pkgconfig
+
+  - name: gsmartcontrol
+    buildsystem: autotools
+    post-install:
+      - install -D host-exec-wrapper.sh /app/bin/host-exec-wrapper
+      - ln -s /app/bin/host-exec-wrapper /app/bin/smartctl
+      - ln -s /app/bin/host-exec-wrapper /app/bin/lsusb
+      - ln -s /app/bin/host-exec-wrapper /app/bin/lspci
+      - ln -s /app/bin/host-exec-wrapper /app/bin/pkexec
+      - ln -s /app/bin/host-exec-wrapper /app/bin/gksu
+      - ln -s /app/bin/host-exec-wrapper /app/bin/gksudo
+      - ln -s /app/bin/host-exec-wrapper /app/bin/kdesu
+      - ln -s /app/bin/host-exec-wrapper /app/bin/kdesudo
+
+      - install -Dm644 docs/assets/images/favicon.svg "{FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg"
+
+    sources:
+      - type: git
+        url: 'https://github.com/ashaduri/gsmartcontrol.git'
+        commit: 49b3d2b9926a446b00970fadac9c52343a5b76bf
+        branch: v1.1.4
```

```suggestion
        tag: v1.1.4
```

a tag isn't a branch